### DISCUSSION TITLE
The point system

### DISCUSSION THREAD

**Discussion by MaelVe** [MediatedCommunications/Mime-Detective#4] [Q&A] [answered]
Hello ! Thank you for your work.
I was wondering how to use the point system (FileExtensionMatch.Points). I suppose it's how much we can trust the file. 
From how many points the file is trusted ?

Thank you !

**Comment by TonyValenti** [comment, marked as answer]
If there are any points at all, it is a valid match. The points help rank overlapping matches.

For example, a Microsoft word document (DOCX) is a type of zip file. If you were to do a detection, you would likely see a high-ranking DOCX match and a low ranking ZIP match. They are both valid matches, but DOCX is better.￼￼￼

**Reply by MaelVe** [reply]
ah okay ! Thank you for the answer !
